### PR TITLE
Add 1.15-stable jobs

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_1_15_stable.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - schedule_failed_builds
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - 2.0.0
+            - 2.1
+            - 2.2
+            - 2.3
```
Comment (domcleal) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_1_15_stable.yaml]: Missing 2.4, compared to develop.

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_15_stable.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - schedule_failed_builds
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+          - 2.0.0
+          - 2.1
+          - 2.2
+          - 2.3
```
Comment (domcleal) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_15_stable.yaml]: Missing 2.4, compared to develop.

**Comment by dLobatog** [comment]
Updated - sorry, I copied them over from 1.14 instead of develop

**Comment by domcleal** [comment]
Merged, thanks @dLobatog.

**Pull request merged**